realm/realm-library/src/main/cpp/io_realm_internal_LinkView.cpp

```diff
@@ -19,10 +19,13 @@
 
 using namespace realm;
 
+static void finalize_link_view(jlong ptr);
+
 JNIEXPORT void JNICALL Java_io_realm_internal_LinkView_nativeClose
```

Comment:
getting the LinkView uses dynamic allocation `new LinkViewRef` I'm not sure the GC will be able to close/delete the reference without an explicit call to close, If it's the case (using `finalize_link_view`) I'm ok with the refactor.